doc/Customization-guide/How-to-contribute/Documentation-contribution-and-style-guide/index.md

```diff
@@ -11,62 +11,168 @@ sort_idx: 30
+
+## Style and tone
+* Use _please_ and _thank you_ judiciously.
+Avoid using _please_ in instructions unless some step in the procedure causes inconvenience for the user or represents a workaround for some system limitation.
+* Use _sorry_ only in error messages that result in serious problems for the user.
+Don’t use _sorry_ if the problem occurred during the normal functioning of the program, such as when the user needs to wait for a network connection to be found.
+* Don’t try to be funny.
+Jokes, slang, and sarcasm are context-specific and hard to translate and localize.
+What’s funny to you might offend or alienate some portion of your audience, so it’s best to avoid these rhetorical ­approaches.
+* Avoid overusing subjective terms such as _easy_, _fast_, _fun_, and so on.
+Instead, demonstrate that something is easy or fun by providing relevant information.
+* Use everyday words when you can, and avoid formal language that you wouldn’t use when speaking to someone in person.
 * Avoid inventing new words or assigning words an unusual meaning.
-* Avoid using please in instructions unless some step in the procedure causes inconvenience for the user or represents a workaround for some system limitation.
-
-## Headings and capitalization
+* Use short, plain words as much as possible.
+* Don’t make generalizations about people, countries, regions, and cultures, especially if the generalizations could be considered derogatory.
+* Avoid using culturally sensitive terms.
+* Omit needless words.
+Don’t use two or three words when one will do.
+* Use questions sparingly. While questions may support a friendlier tone, overuse of questions doesn’t support a trustworthy voice.
+Questions can work well when users actually do have them, not when we invent them on their behalf.
+* Use exclamation points sparingly.
+The overuse of exclamation points can make content seem effusive or ingratiating.
+Save exclamation points for when they count.
+* Be mindful of globalization and cultural considerations before you use colloquialisms or idioms.
+* Do not use non-English words or phrases, such as _de facto_ or _ad hoc_, in English content, even if you think these terms are generally known and understood.
+
+## Use of terminology
+* Whenever possible, you should get your point across by using common English words.
+It is all right to use technical terms when they are necessary for precise communication, even with a general audience, but do not write as if everybody understands these terms or will immediately grasp their meaning.
+* Define terms in the text as you introduce them.
+* Spell out an acronym on its first mention on the page, for example, _Common Type Library (CTL)_.
+* Provide links from the main text to the glossary.
+* Do not create a new term if a term describing a concept already exists in the glossary.
+* If you need to create a new term, verify that the term that you select is not already in use to mean something else.
+
```

Comment:
When using defined terms, make sure you use them exactly as defined. Shortcutting only works when it's defined. Examples: "Administrative UI" is not equal to "Admin UI" unless defined so; "tenant administrator" is not the same as "tenant", etc.